sdk/remoterendering/mixedreality-remoterendering/README.md

```diff
@@ -0,0 +1,408 @@
+
+In all the following examples, the client is constructed with a `remoteRenderingEndpoint` Uri object.
+The available endpoints correspond to regions, and the choice of endpoint determines the region in which the service performs its work.
+An example is `https://remoterendering.eastus2.mixedreality.azure.com`.
+
+> NOTE: For converting assets, it is preferable to pick a region close to the storage containing the assets.
+
+> NOTE: For rendering, it is strongly recommended that you pick the closest region to the devices using the service.
+> The time taken to communicate with the server impacts the quality of the experience.
+
+#### Authenticating with account key authentication
+
+Use the `AccountKeyCredential` object to use an account identifier and account key to authenticate:
+
+```typescript Snippet:CreateAClient
+const credential = new AzureKeyCredential(accountKey);
+
+const client = new RemoteRenderingClient(serviceEndpoint, accountId, accountDomain, credential);
+```
+
+#### Authenticating with an AAD client secret
+
+Use the `ClientSecretCredential` object to perform client secret authentication.
+
+```typescript Snippet:CreateAClientWithAAD
+let credential = new ClientSecretCredential(tenantId, clientId, clientSecret, {
+  authorityHost: "https://login.microsoftonline.com/" + tenantId
+});
+
+const client = new RemoteRenderingClient(serviceEndpoint, accountId, accountDomain, credential);
+```
+
+#### Authenticating a user using device code authentication
+
+Use the `DeviceCodeCredential` object to perform device code authentication.
+
+```typescript Snippet:CreateAClientWithDeviceCode
+let deviceCodeCallback = (deviceCodeInfo: DeviceCodeInfo) => {
+  console.debug(deviceCodeInfo.message);
+  console.log(deviceCodeInfo.message);
+};
+
+let credential = new DeviceCodeCredential(tenantId, clientId, deviceCodeCallback, {
+  authorityHost: "https://login.microsoftonline.com/" + tenantId
+});
+
+const client = new RemoteRenderingClient(serviceEndpoint, accountId, accountDomain, credential);
+```
+
+See [here](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/Device-Code-Flow) for more
+information about using device code authentication flow.
+
+#### Interactive authentication with DefaultAzureCredential
+
+Use the `DefaultAzureCredential` object with `includeInteractiveCredentials: true` to use default interactive authentication
+flow:
+
+```typescript Snippet:CreateAClientWithAzureCredential
+let credential = new DefaultAzureCredential();
+
+return new RemoteRenderingClient(serviceEndpoint, accountId, accountDomain, credential, {
+  authenticationEndpointUrl: "https://sts.mixedreality.azure.com"
+});
+```
+
+#### Authenticating with a static access token
+
+You can pass a Mixed Reality access token as an `AccessToken` previously retrieved from the
+[Mixed Reality STS service](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/mixedreality/Azure.MixedReality.Authentication)
```

Comment:
```suggestion
[Mixed Reality STS service](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/mixedreality/Azure.MixedReality.Authentication)
```

Reply:
Wait, this is a .NET thing. Please make sure everything here is JS-specific.

Reply:
Good catch. Thanks.